sv_pipeline/genome/load_data.py

```diff
@@ -129,20 +142,25 @@ def annotate_fields(mt, gencode_release, gencode_path):
 
     gene_id_mapping = hl.literal(load_gencode(gencode_release, download_path=gencode_path))
 
-    rows = rows.annotate(
-        sortedTranscriptConsequences=hl.flatmap(lambda x: x, hl.filter(
+    rows = rows.annotate(**{
+        SORTED_TRANS_CONSEQ: hl.flatmap(lambda x: x, hl.filter(
             lambda x: hl.is_defined(x),
-            [rows.info[col].map(lambda gene: hl.struct(gene_symbol=gene, gene_id=gene_id_mapping[gene],
-                                                       major_consequence=col.split('__')[-1]))
+            [rows.info[col].map(lambda gene: hl.struct(**{'gene_symbol': gene, 'gene_id': gene_id_mapping[gene],
+                                                       MAJOR_CONSEQ: col.split('__')[-1]}))
              for col in [gene_col for gene_col in rows.info if gene_col.startswith('PROTEIN_CODING__')
                          and rows.info[gene_col].dtype == hl.dtype('array<str>')]])),
-        sv_type=rows.alleles[1].replace('[<>]', '').split(':', 2),
+        SV_TYPE: rows.alleles[1].replace('[<>]', '').split(':', 2)}
     )
 
     DERIVED_FIELDS.update({'filters': lambda rows: hl.if_else(hl.len(rows.filters) > 0, rows.filters,
                                                                  hl.missing(hl.dtype('array<str>')))})
     rows = rows.annotate(**{k: v(rows) for k, v in DERIVED_FIELDS.items()})
 
+    rows = rows.annotate(**{v: get_sample_in_gq_range(rows, i, i+QS_BIN_SIZE) for i, v in SAMPLE_QS_FIELDS.items()})
+
+    rows = rows.annotate(**{TRANS_CONSEQ_TERMS: rows[TRANS_CONSEQ_TERMS]
```

Comment:
It is much cleaner to just modify the function for this field in `DERIVED_FIELDS`